exercises/ex1.cpp

```diff
@@ -5,44 +5,43 @@ using namespace std;
 
 int main()
 {
-  int week;
-
-  /* Input week number from user */
-  cout << "Enter week number(1-7): " << endl;
-  cin >> week;
-
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
-
-  return 0;
-}
\ No newline at end of file
+    int week;
+    /* Input week number from user */
+    cout << "Enter week number(1-7): " << endl;
+    cin >> week;
+    switch (week) {
+    case 1:
+      cout << "Monday" << endl;
+      break;
+
+    case 2:
+      cout << "Tuesday" << endl;
+      break;
+
+    case 3:
+      cout << "Wednesday" << endl;
+      break;
+
+    case 4:
+      cout << "Thursday" << endl;
+      break;
+
+    case 5:
+      cout << "Friday" << endl;
+      break;
+
+    case 6:
+      cout << "Saturday" << endl;
+      break;
+
+    case 7:
+      cout << "Sunday" << endl;
+      break;
+
+    default:
+      cout << "Invalid input! Please enter week number between 1-7." << endl;
+      break;
+    }
+
```

Comment:
L'istruzione `<< endl` è ridondante in tutti i case dello switch. Valuta di inserirne uno solo appena lo switch finisce.